apollo-router-core/src/services/router_service.rs

```diff
@@ -303,18 +290,20 @@ impl PluggableRouterServiceBuilder {
 
         // ExecutionService takes a PlannedRequest and outputs a RouterResponse
         let (execution_service, execution_worker) = Buffer::pair(
-            ServiceBuilder::new()
-                .layer(ForbidHttpGetMutationsLayer::default())
-                .service(
-                    self.plugins.iter_mut().rev().fold(
-                        ExecutionService::builder()
-                            .schema(self.schema.clone())
-                            .subgraph_services(subgraphs)
-                            .build()
-                            .boxed(),
-                        |acc, e| e.execution_service(acc),
+            ForbidHttpGetMutationsLayer::default()
```

Comment:
Nit: Can we keep service builder here? We will likely add more layers and it makes it clear that we are building a service.

Reply:
fixed in ee986a9d